estela_cli/deploy.py

```diff
@@ -40,7 +40,7 @@ def zip_project(pid, project_path):
                 zip.write(filename, arcname)
 
 
-def update_dockerfile(requirements_path, python_version):
+def update_dockerfile(requirements_path, python_version, repository_entrypoint, branch):
     dockerfile_path = get_estela_dockerfile_path()
```

Comment:
Here is not necessary the variable branch. Instead repository_entrypoint, just entrypoint is a better name.